akka-remote/src/main/scala/akka/remote/artery/tcp/ssl/X509Readers.scala

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.remote.artery.tcp.ssl
+
+import java.security.cert.X509Certificate
+import java.util
+
+import akka.annotation.InternalApi
+import javax.naming.ldap.LdapName
+import akka.util.ccompat.JavaConverters._
+
+/**
+ * INTERNAL API
+ */
+@InternalApi
+object X509Readers {
```

Comment:
`private[akka]`

Reply:
I've been using the more strict `private[tcp]` or even `private[ssl]` in some places. Should I fallback to `private[akka]` elsewhere too?

Reply:
Doesn't matter much. I often run into more trouble than what it's worth when using more strict private packages. One exposing something else in method and then compiler warns about visibility.